cmd/manager/main.go

```diff
@@ -154,6 +155,11 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err := consolev1.AddToScheme(mgrScheme); err != nil {
+		log.Error(err, "Failed adding openshift/api/console/v1 to scheme")
```

Comment:
Nit: "Failed to add..."

Reply:
I used the same formatting as above scheme additions. I'd keep it the same way, later we could refactor all of them